documentation/book/con-requests-kafka-bridge.adoc

```diff
@@ -56,9 +67,11 @@ When creating a consumer using the `/consumers/_groupid_` endpoint, the `POST` r
 
 <1> A binary embedded data format.
 
+If an embedded data format for the consumer is not specified, then a binary format is set.
+
 The embedded data format specified when creating a consumer must match the data format of the Kafka messages it will consume. 
 
-If you choose to specify a binary embedded data format, subsequent producer requests must provide the binary data in the request body as Base64-encoded strings. For example, when sending messages using the `/topics/_topicname_` endpoint, `records.value` must be encoded in Base64:
+If you choose to specify a binary embedded data format, subsequent producer requests must provide the binary data in the request body as Base64-encoded strings. For example, when sending messages by making `POST` requests to the `/topics/_topicname_` endpoint, the `value` must be encoded in Base64:
```

Comment:
for example here you didn't remove the "_" in topicname, I think that we should use placeholders with braces `/topics/{topicname}`

Reply:
Changed.